### PR TITLE
Added annotation to example task & README

### PR DESCRIPTION
**Pull request by mjbrender**
* Added  which collects load statistics from psutil and publishes them to.. you guessed it
* Heavily annotated the task above to walk people through an example. We could (and should) do more like it
* Added some context to the tasks folder README
* Included links to Roadmap specs

### CODE REVIEW

File: examples/tasks/README.md [hunk 1]
```diff
@@ -27,16 +30,24 @@ limitations under the License.
...
   - schedule
     - interval (1s)
   - collector
-    - CEPH
+    - Ceph
   - publisher
     - file
 
-- **psutil-influx.json**
+- **psutil-influx.json**: a more complex sexample that collects information from psutil and publishes to an instance of InfluxDB running locally. See [influxdb-grafana](../influxdb-grafana/) for other files to get this running.
```
Comment (tiffanyfj) [inline examples/tasks/README.md]: s/sexample/example

**Comment by mjbrender** [comment]
Good catch :ok_hand:. Fixed and rebased.

**Comment by mjbrender** [comment]
@tiffanyfj mind a quick double check when you can?

File: examples/tasks/psutil-file.yaml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+--- # The Header starts after this line
+  version: 1        # All tasks, like plugins, have versioning
+  schedule:         # How frequently should we run this task?
+    type: "simple"  # Simple means run forever at the interval below
+    interval: "1s"  # This accepts seconds ('s') and milliseconds  ('ms')
+  workflow:           # For a small number of statistics, you can go as small
+    collect:          # as 500ms. For a large number, more than 1s may be needed
```
Comment (tiffanyfj) [inline examples/tasks/psutil-file.yaml]: I would need to check or get someone else's feedback, but I don't know that 500ms is really the smallest. I think it's just that the current cache limit is 500ms.

**Comment by mjbrender** [comment]
Thanks @tiffanyfj - updated based on talks w/ @danielscottt. TL;DR is we've tested down to 10ms and cache at a default of 500ms.

**Comment by danielscottt** [comment]
aside from YASTF (yet another spurious travis failure), is this ready to go @mjbrender?

**Comment by mjbrender** [comment]
@danielscottt yup - just rebased since @geauxvirtual's updates to get this on track. Good to merge once tests complete :+1:

**Comment by danielscottt** [comment]
LGTM

:ship:

**Pull request merged**